src/test/java/edu/ucla/library/avpairtree/verticles/WatcherVerticleTest.java

```diff
@@ -113,15 +113,14 @@ public void testWatcherAudioCsvParsing(final TestContext aContext) {
 
             // Send a message with the CSV file location to the watcher verticle
             return vertx.eventBus().<CsvItem>request(WatcherVerticle.class.getName(), csvFilePath);
-        }).compose(result -> {
-            return checkOutput(csvFilePath.replace(CSV_EXT, OUT_EXT), expectedUpdates, aContext);
-        }).onComplete(check -> {
-            if (check.succeeded()) {
-                asyncTask.countDown();
-            } else {
-                aContext.fail(check.cause());
-            }
-        }).onFailure(error -> aContext.fail(error));
+        }).compose(result -> checkOutput(csvFilePath.replace(CSV_EXT, OUT_EXT), expectedUpdates, aContext))
+                .onComplete(check -> {
+                    if (check.succeeded()) {
+                        asyncTask.countDown();
+                    } else {
+                        aContext.fail(check.cause());
+                    }
+                }).onFailure(error -> aContext.fail(error));
```

Comment:
I don't like this formatting since the indentation indicates to me that the calls to `onComplete` and `onFailure` are called within the function passed to `compose`. The call to `checkOutput` is also kind of long, and I think it's more difficult to quickly scan that line now.

Reply:
Could this code be reverted to how it was previously without upsetting our style checks?

Reply:
No, the code is formatted now as a part of the build process. We can open a ticket to fix that in the formatter though.